components/Shell/Aside.tsx

```diff
@@ -13,41 +13,38 @@ import {
 
 import { baseUrls } from 'data';
 import { Divider } from 'components/Divider';
-import { useRouter } from 'next/router';
 
 export type AsideItem = {
   id: string;
   title: string;
   level: number;
 };
 
-export const Aside = ({ asideItems }: { asideItems: AsideItem[] }) => {
+export const Aside = ({
+  asideItems,
+  path
+}: {
+  asideItems: AsideItem[];
+  path: string;
+}) => {
   const [scrollPosition, setScrollPosition] = useState(0);
 
+  // Edit this page URL
+  const baseEditPath = 'https://github.com/icerpc/icerpc-docs/tree/main/pages';
+  let basePath = path.split('#')[0];
+  basePath = resolvePath(path);
+  basePath = path.replace(/^\/slice\d/, '/slice'); // strip away slice version
+  if (!basePath.endsWith('.md')) basePath += '.md'; // ensure that basePath ends with .md
```

Comment:
We should be able to always add on .md right? I don't think there's a case where it's already there.